ringct.md

```diff
@@ -17,51 +17,50 @@ This is accomplished by the digital signature, which can only be proven to be si
-
-Beginning in mid-January 2017, RingCT became the default type of Monero transaction, and only a month later, over 98% of new transactions used RingCT.
 RingCT prevents third parties from knowing how much Monero is being transacted.
+
+Beginning in January 2017, RingCT became the default type of Monero transaction, and only a month later, over 98% of new transactions used RingCT.
 After September 2017, the use of RingCT will be mandatory for all Monero transactions.
 
 Now, let’s walk through an example to see how these concepts tie together.
 
 ## 4) – Ring Signatures Transaction
 
-All the newly created monero firstly reside in outputs with visible amount, so everyone can see the supply.
+All the newly created monero firstly reside in outputs with a visible amount, so everyone can see the supply.
 First time a newly created monero is moved, that transaction creates outputs with masked amounts.
-Similarly, any old output must be converted to RCT first after which it can be combined in a ring signature with any other RCT output.
+Similarly, any old output must be converted to an RCT output first before it can be combined in a ring signature with any other RCT output.
 Consequently, transactions no longer have to be broken down into different denominations as before.
 This means that your wallet is free to pick ring partners from all RCT outputs, significantly improving privacy.
 Let's take a deeper dive to see how a RingCT transaction works.
 
 Imagine you have a RCT input of 12.56 Monero and would like to send 2.5 Monero to your friend.
 The transaction will then have one input of 12.56, and 2 outputs: one of 2.5 monero to your friend, and one of 10.06 back to your wallet as change output.
-Since monero outputs can't be spent from 2 times, you must always spend it entirely and return the change to yourself.
+Since monero outputs can't be spent twice, you must always spend it entirely and return the change to yourself.
 With this input, you cryptographically commit to some amount without revealing it.
 With RCT, instead of revealing the values 12.56, 10.06, and 2.5, you cryptographically commit to them.
 The commitment will look like random numbers, but they actually have important features which ensures safety of the mechanism.
-To verify that no new money is created, the sum of inputs minus the sum of outputs must be 0.
+To verify that no new money is created, the sum of inputs must equal the sum of the outputs.
 This is easy to check if you see the numbers.
 With commitments, it can be checked exactly the same way!
-The only difference is that you're summing some seemingly random numbers, but the sum must again be 0.
-The underlying cryptography guarantees that if the sum of commintment is 0, then also the sum of underlying masked values must be 0.
+The only difference is that you're comparing seemingly random numbers, but they must still will be the same on both sides of the transaction.
```

Comment:
Can we reword to: The only difference is that you're comparing seemingly random numbers, but they can still be summed to check both sides of the transaction.